contracts/IntervalTokenStore.sol

```diff
@@ -0,0 +1,142 @@
+// IntervalTokenStore stores tokens for applications, which have discrete states increasing with time
```

Comment:
This should go right above the contract definition. I'd also reword it a bit. Maybe call it IntervalTokenLocker or EpochTokenLocker instead of Store?

```
Allows to deposit token which become credited in the next epoch and allows to request a token withdraw which becomes claimable after the current epoch expired.
```